pkgs/applications/social/patchwork/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, appimage-run, fetchurl, runtimeShell, gsettings-desktop-schemas, gtk3, gobject-introspection, wrapGAppsHook }:
+
+let
+  version = "3.11.6";
+in
+
+stdenv.mkDerivation rec {
```

Comment:
I think you should use `appimageTools`, take a look at #54693.

Reply:
@bricewge i've mirrored other examples so i don't think this is needed

https://github.com/NixOS/nixpkgs/pull/54693 my understanding of the conversation here is that `appimageTools` referenced in that PR is that it is for the build side, not the download and run side